src/packages/cli/package.json

```diff
@@ -110,7 +110,7 @@
     "typescript": "4.3.2"
   },
   "scripts": {
-    "test:commands": "./fixtures/test.sh && jest --maxConcurrency=1",
+    "test:commands": "./fixtures/test.sh && jest --maxConcurrency=1  --forceExit",
```

Comment:
Meh, that is not good. Let's create an issue to track that we need to remove this (and other instances we added) later as it indicates something is not finishing as it should.

Reply:
I added this information to this [PR](https://github.com/prisma/prisma/pull/7680#issuecomment-863936516), which @madebysid will be working on